### DISCUSSION TITLE
Accessibility for colorblind people

### DISCUSSION THREAD

**Discussion by michauxb** [magicbug/Cloudlog#2982] [Ideas / Feature Requests]
Hi folks,
I'm colorblind (deuteranopia) and I noticed that it's really difficult to see in the logbook the status of the QSL/eQSL. Indeed, you're using red, green and a kind of orange. It could be very usefull for accessibility to add a checkbox to adapt these little arrows. It's a stupid feature, but it could be very cool.

**Comment by phl0** [comment]
> Hi folks, I'm colorblind (deuteranopia) and I noticed that it's really difficult to see in the logbook the status of the QSL/eQSL. Indeed, you're using red, green and a kind of orange. It could be very usefull for accessibility to add a checkbox to adapt these little arrows. It's a stupid feature, but it could be very cool.

We did add an option to color the markers by a custom theme. I just need to re-find the stuff ...

**Reply by phl0** [reply]
Ah here it is. Tnx @AndreasK79.

https://github.com/magicbug/Cloudlog/issues/1787
